src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -123,6 +136,22 @@ private ScanResults Scan_Integration_Core(string testAppPath, int expectedErrorC
             return output;
         }
 
+        private ScanResults ScanWithProvisionForBuildAgents(IScanner scanner)
+        {
+            try
+            {
+                return scanner.Scan();
+            }
+            catch (AxeWindowsAutomationException e)
+            {
+                if (allowInconclusive && e.Message.Contains("Automation017"))
+                {
+                    Assert.Inconclusive("Unable to complete Integration tests");
+                }
```

Comment:
Shouldn't there be a return statement here to return before throwing the exception?

Reply:
Not needed. Assert.Inconclusive ends the test. According to https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.assert.inconclusive?view=mstest-net-1.2.0, AssertInconclusive is a wrapper than throws an AssertInconclusiveException, when then gets handled by the framework.